x/delayedack/rollapp_hooks.go

```diff
@@ -61,8 +61,8 @@ func (im IBCMiddleware) finalizeRollappPacket(
 ) (err error) {
 	logger.Debug("Finalizing IBC rollapp packet",
 		"rollappID", rollappID,
-		"sequence", rollappPacket.Packet.GetSequence(),
-		"destination channel", rollappPacket.Packet.GetDestChannel(),
+		"sequence", rollappPacket.Packet.Sequence,
+		"source channel", rollappPacket.Packet.SourceChannel,
```

Comment:
why logging source only? for `OnRecv` the hub's channel is the dest
anyway the logging data can be squeezed into logging `PacketUID`

Reply:
as discussed, because we use the source for the packetKey so will be easier to track which packet key is the problematic.

Reply:
prettified logging and added both dest and source